scripts/docker-integration-tests/run.sh

```diff
@@ -23,9 +23,15 @@ MAX_IDX=$(((NUM_TESTS*(BUILDKITE_PARALLEL_JOB+1)/BUILDKITE_PARALLEL_JOB_COUNT)-1
 ITER=0
 for test in "${TESTS[@]}"; do
 	if [[ $ITER -ge $MIN_IDX && $ITER -le $MAX_IDX ]]; then
+		# Ensure all docker containers have been stopped so we don't run into issues
+		# trying to bind ports.
+		CONTAINERS=$(docker ps -aq)
+		if [[ -n "$CONTAINERS" ]]; then
```

Comment:
i'd actually replace this whole thing with

```bash
docker rm -f $(docker ps -aq) 2>/dev/null || true
```

for brevity, but mostly just because the fewer lines of bash we have, the better.